### PR TITLE
[Office365] Qol improvements.

### PR DESCRIPTION
**Pull request by ngodfraind**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Fixed tickets |  #983

see #983 

I can not test it.

### CODE REVIEW

**Comment by LaurentGruber** [comment]
LGTM

**Comment by purplefish32** [comment]
LGTM

File: plugin/o365/Resources/views/Authentication/missingProperties.html.twig [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{% extends 'ClarolineCoreBundle:Exception:layout.html.twig' %}
+
+{% set httpCode = 400 %}
```
Comment (stefk) [inline plugin/o365/Resources/views/Authentication/missingProperties.html.twig]: 400 is for HTTP syntax errors. Data validation is more or less covered by 422.

**Comment by stefk** [comment]
@ngodfraind, please use accurate commit messages / PR titles. Things "Qol" or "Estp" have no meaning for most (including me) and shouldn't appear in the commit log except as sidenotes IMO.

**Comment by ngodfraind** [comment]
Qol is Quality of life acronym ;). Estp doesn't make sense that's true but I think I already renamed it.

**Comment by LaurentGruber** [comment]
LGTM

**Pull request merged**